sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -725,7 +726,7 @@ def __init__(self, key_coder_impl, window_coder_impl):
     self._tag_coder_impl = StrUtf8Coder().get_impl()
 
   def encode_to_stream(self, value, out, nested):
-    # type: (dict, create_OutputStream, bool) -> None
+    # type: (userstate.Timer, create_OutputStream, bool) -> None
```

Comment:
as far as I can tell, this was just wrong

Reply:
I think it used to be correct back when timers were being implemented. This code changed a couple of months ago too.